cli/test/lib/exec/spawn_spec.js

```diff
@@ -154,6 +154,16 @@ describe('lib/exec/spawn', function () {
       })
     })
 
+    context('closes', function () {
```

Comment:
what is exit is fired and then close is fired - should it be tested?

Reply:
good point, added one